### ISSUE TITLE
NDK Version update Android appcenter build

### ISSUE THREAD

**Issue by D3laru3** [microsoft/appcenter#1740]
Is it possible to update NDK to the latest version? I'm experiencing an issue with the appcenter build due to NDK STL shared object file. 

`Expected NDK STL shared object file at /Users/runner/Library/Android/sdk/ndk-bundle/sources/cxx-stl/gnu-libstdc++/4.9/libs/armeabi-v7a/libgnustl_shared.so`

The issue might be connected with the DexGuard which we have to include in our project. It looks like DexGuard using the 18+ version of the NDK

I tried to set android.ndkVersion in our Gradle to version. 21..+. Unfortunately, local version of NDK in appcenter is set to 18.1.5063045.

`No version of NDK matched the requested version 21.0.6113669. Versions available locally: 18.1.5063045, 18.1.5063045`

Thanks

**Comment by winnie** [comment]
Thanks for filing an issue. Tagging our Build PM @elamalani to address this. Thanks!

**Comment by MaxenceMax** [comment]
Does this issue fixed ?

**Comment by elamalani** [comment]
https://github.com/microsoft/appcenter/issues/1728

**Comment by dacheng81** [comment]
Just to chime in here: on Azure Devops, their VMs support `21.0.6113669`. I can't remember if all 3 platforms (Windows, Mac, Linux) support it. I know their Ubuntu VM did not support 20.x version.

**Comment by flamewave000** [comment]
Definitely need this upgraded. Gradle 4.0.0 has been released, and I'm still stuck on 3.5.3 because AppCenter is breaking my entire deployment process if I upgrade. My company has started considering alternative CI/CD solutions due to this specific issue.

**Comment by NasarIqbalTechswivel** [comment]
This issue can also solve by adding `android {
    ndkVersion "major.minor.build" // e.g.,  ndkVersion '21.3.6528147'
}` in gradel build. you can see [here ](https://developer.android.com/studio/projects/install-ndk#apply-specific-version)

**Comment by yasirtrimulabs** [comment]
Any update on this?
I'm getting a similar error on app center 
```
Execution failed for task ':app:stripReleaseDebugSymbols'.
> No version of NDK matched the requested version 21.3.6528147. Versions available locally: 18.1.5063045, 21.4.7075529, 21.4.7075529, 21.4.7075529, 22.1.7171670, 23.1.7779620
```

**Comment by stephan-markovic** [comment]
Closing as duplicate of https://github.com/microsoft/appcenter/issues/1728
